client/scripts/views/pages/view_proposal/body.ts

```diff
@@ -287,6 +285,230 @@ export const ProposalBodyDeleteMenuItem: m.Component<{
   }
 };
 
+export const EditPermissionsButton: m.Component<{
+  openEditPermissions: Function,
+}> = {
+  view: (vnode) => {
+    const { openEditPermissions } = vnode.attrs;
+    return m(MenuItem, {
+      label: 'Manage collaborators',
+      onclick: async (e) => {
+        e.preventDefault();
+        openEditPermissions();
+      }
+    });
+  }
+};
+
+export const ProposalEditorPermissions: m.Component<{
+  thread: OffchainThread,
+  popoverMenu: boolean,
+  onChangeHandler: any,
+  openStateHandler: any
+}, {
+  membersLoaded: boolean,
+  items: any[],
+  addedEditors: any,
+  removedEditors: any,
+  isOpen: boolean,
+}> = {
+  view: (vnode) => {
+    const { thread } = vnode.attrs;
+    if (!vnode.state.membersLoaded) {
+      // TODO: Break into view
+      const chainOrCommObj = app.chain ? { chain: app.activeChainId() } : { community: app.activeCommunityId() };
```

Comment:
This will run multiple times when the view is rerendered before the request returns.

Reply:
Fixed with a "membersFetched" check, since vnode.state.items.length will check that members have finished fetching.